opencensus/stats/internal/view_data_impl.cc

```diff
@@ -93,7 +94,17 @@ ViewDataImpl::ViewDataImpl(const ViewDataImpl& other, absl::Time now)
                 : Type::kDouble),
       start_time_(std::max(other.start_time(),
                            now - other.aggregation_window().duration())) {
+  // Intentionally reset the source with a new start time.
+  new (&start_times_) DataMap<absl::Time>();
+  for (const auto& it : other.start_times_) {
+    auto new_start_time =
+        std::max(it.second, now - other.aggregation_window().duration());
+    start_times_[it.first] = new_start_time;
+  }
+
+  new (&start_times_) DataMap<absl::Time>(other.start_times_);
```

Comment:
Is this clearing start_times_?

Reply:
This is a mistake. I believe this is the correct logic now. Matching the behaviour of the original start_time_